fala/playwright/setup.py

```diff
@@ -16,3 +18,59 @@ def tearDownClass(cls):
         super().tearDownClass()
         cls.browser.close()
         cls.playwright.stop()
+
+    def visit_results_page(self, postcode, checkbox_labels=None):
```

Comment:
these three methods for visiting the results pages are all very similar - could we create one method? Or two at most for the other regions if you think that is more readable.

Reply:
Have kept to two for readability.

Reply:
let's revisit this too